sdk/Sdk.Generators/FunctionMetadataProviderGenerator.Emitter.cs

```diff
@@ -5,16 +5,20 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
+using System.Text.Json;
 using System.Threading;
 using Microsoft.CodeAnalysis;
-using Microsoft.CodeAnalysis.Text;
 
 namespace Microsoft.Azure.Functions.Worker.Sdk.Generators
 {
     public partial class FunctionMetadataProviderGenerator
     {
         internal sealed class Emitter
         {
+            JsonSerializerOptions _jsonOptions = new JsonSerializerOptions
+            {
+                PropertyNamingPolicy = JsonNamingPolicy.CamelCase
```

Comment:
To make the world a happier place, also use: `DictionaryKeyPolicy = JsonNamingPolicy.CamelCase,`

Reply:
This is what I needed!! Thank you - I think it removes the need for the lowercasing logic discussed offline but will validate first.